### PR TITLE
trying this out

### PR DESCRIPTION
**Pull request by jonfreihofer**
Spent some time trying to do this right, but its broken at the moment. Won't render I haven't built the extra components yet but I can't quite figure out what I did wrong, hopefully nothing too crazy. Now that I think of it I'm not sure if creating a PR is the right move for that scenario, haha. oops? maybe its

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (jonwalz) [inline .vscode/launch.json:1]: Looks like we need to add the `.vscode` folder to the `.gitignore` file. This file doesn't need to be committed to github

File: NavBar.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from "react"
+
+
+
```
Comment (jonwalz) [inline NavBar.js]: We can remove these white spaces

File: index.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import React from "react"
+import ReactDom from "react-dom"
+import NavBar from "./NavBar"
+
+
+
```
Comment (jonwalz) [inline index.js]: We can remove these white spaces

File: index.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import React from "react"
+import ReactDom from "react-dom"
```
Comment (jonwalz) [inline index.js]: Spelling error. `ReactDom` should be `ReactDOM`

File: react-parcel/package.json [hunk 5]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (jonwalz) [inline react-parcel/package.json]: We don't need this `react-parcel` directory. Not sure where it came from

**Review by jonwalz — CHANGES_REQUESTED** [review]
Just the spelling error and remove the `react-parcel` directory.

**Review by jonfreihofer — COMMENTED** [review]
made the requested changes

**Comment by jonwalz** [comment]
Oops, look like we need to add `.cache` to the `.gitignore` as well

**Comment by jonwalz** [comment]
@jonfreihofer Awesome work! I'm stoked by how quickly you're learning this stuff. The commit I added is just to remove the cache files from git history 🚀

**Pull request merged**